README.rst

```diff
@@ -193,72 +192,70 @@ If you want to run local integration tests, then use:
-                      --processor cpu
+    pytest test/integration/local --docker-base-name preprod-mxnet \
+                                  --tag 1.0 \
+                                  --py-version 2 \
+                                  --framework-version 0.12.1 \
+                                  --processor cpu
 
-Functional Tests
-~~~~~~~~~~~~~~~~
+SageMaker Integration Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Functional tests require your Docker image to be within an `Amazon ECR repository <https://docs
+SageMaker Integration tests require your Docker image to be within an `Amazon ECR repository <https://docs
 .aws.amazon.com/AmazonECS/latest/developerguide/ECS_Console_Repositories.html>`__.
 
-The Docker-base-name is your `ECR repository namespace <https://docs.aws.amazon
+The Docker base name is your `ECR repository namespace <https://docs.aws.amazon
 .com/AmazonECR/latest/userguide/Repositories.html>`__.
 
-The instance-type is your specified `Amazon SageMaker Instance Type
-<https://aws.amazon.com/sagemaker/pricing/instance-types/>`__ that the functional test will run on.
+The instance type is your specified `Amazon SageMaker Instance Type
+<https://aws.amazon.com/sagemaker/pricing/instance-types/>`__ that the tests will run on.
 
-Before running functional tests:
+Before running SageMaker Integration tests:
 
 #. Build your Docker image.
 #. Push the image to your ECR repository.
 #. Pass in the correct pytest arguments to run tests on SageMaker against the image within your ECR repository.
 
-If you want to run a functional end to end test on `Amazon
-SageMaker <https://aws.amazon.com/sagemaker/>`__, then use:
+If you want to run an integration test on `Amazon SageMaker <https://aws.amazon.com/sagemaker/>`__, then use:
 
 ::
 
-    # Required arguments for integration tests are found in test/functional/conftest.py
+    # Required arguments for integration tests are found in test/conftest.py
 
-    pytest test/functional --aws-id <your_aws_id> \
-                           --docker-base-name <your_docker_image> \
-                           --instance-type <amazon_sagemaker_instance_type> \
-                           --tag <your_docker_image_tag> \
+    pytest test/integration/sagmaker --aws-id <your_aws_id> \
+                                     --docker-base-name <your_docker_image> \
+                                     --instance-type <amazon_sagemaker_instance_type> \
+                                     --tag <your_docker_image_tag> \
 
 ::
 
     # Example
-    pytest test/functional --aws-id 12345678910 \
-                           --docker-base-name preprod-mxnet \
-                           --instance-type ml.m4.xlarge \
-                           --tag 1.0
+    pytest test/integration/sagemaker --aws-id 12345678910 \
+                                      --docker-base-name preprod-mxnet \
+                                      --instance-type ml.m4.xlarge \
+                                      --tag 1.0
```

Comment:
Let's be consistent with our example and do something like '0.12.1-gpu-py3' here